dojo.py

```diff
@@ -5,7 +5,7 @@
 run the application in interactive mode.
 
 Usage:
-    Dojo CREATE_ROOM <room_name>...
+    Dojo CREATE_ROOM (<room_name> <room_type>)...
```

Comment:
Not all terminals will treat an uppercase command as similar to a lowercase command and most people are accustomed to lowercase commands. It will be good to consider using lowercase commands